web-client/public/js/grnsight.js

```diff
@@ -1,10 +1,17 @@
-const drawGraph = require("./graph").drawGraph;
-const sliderObject = require("./sliders").sliderObject;
-const sliderGroupController = require("./sliders").sliderGroupController;
-const container = require("./container").container;
-const displayStatistics = require("./graph-statistics").displayStatistics; // eslint-disable-line no-unused-vars
-const upload = require("./upload").upload;
-const nodeColoringController = require("./node-coloring").nodeColoringController;
+import { drawGraph } from "./graph";
+import { sliderObject } from "./sliders";
+import { sliderGroupController } from "./sliders";
+import { container } from "./container";
+import { displayStatistics } from "./graph-statistics"; // eslint-disable-line no-unused-vars
+import { upload } from "./upload";
+import { nodeColoringController } from "./node-coloring";
```

Comment:
The transition from `require` to `import` wasn't necessary but I was already modifying this file so I figured it wouldn't be a bad idea to show how to make this change. You'll notice that `import` is less verbose and generally easier to read.